### PR TITLE
Separate namespace

### PR DESCRIPTION
**Pull request by rmohr**
Move Jobs and Pods into a separate namespace, so that jobs are isolated from the prow components and that we can have for instance distinct memory limits.

### CODE REVIEW

File: github/ci/prow/files/config.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ sinker:
   max_prowjob_age: 48h
   max_pod_age: 30m
 
-prowjob_namespace: kubevirt-prow
-pod_namespace: kubevirt-prow
+prowjob_namespace: kubevirt-prow-jobs
+pod_namespace: kubevirt-prow-jobs
```
Comment (slintes) [inline github/ci/prow/files/config.yaml]: wasn't the idea to use different namespaces? :thinking:

Comment (rmohr) [reply github/ci/prow/files/config.yaml]: Haha, for now different than the prow components.

Comment (slintes) [reply github/ci/prow/files/config.yaml]: got it ;) :+1:

**Pull request merged**